manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+To get to Kubeflow Pipelines page in Marketplace, enter "Kubeflow Pipelines" in the search box in Marketplace landing page. 
+Alternatively you can access the [Marketplace page for Kubeflow Pipelines](https://console.cloud.google.com/marketplace/details/google-cloud-ai-platform/kubeflow-pipelines) directly.
+
+You can find Kubeflow Pipelines instances deployed via Marketplace in [Pipelines Console](http://console.cloud.google.com/ai-platform/pipelines).
+
+## Cluster
+
+You can choose an existing cluster for Kubeflow Pipelines deployment, or create a new cluster. New cluster created from Marketplace config page cannot be customized.
+If you want to customize the cluster, please go to [Google Kubernetes Engine](https://console.cloud.google.com/kubernetes/list) to create one.
+
+Please notice that you can only `install one Kubeflow Pipelines into one cluster`.
+
+## Namespace
+Please specify a [Kubenetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+## App instance name
+It is possible to install multiple instances of Kubeflow Pipelines in a single cluster.
```

Comment:
Because it is possible to deploy multiple instances of Kubeflow Pipelines in a single cluster, specify an app instance name to help you identify this instance.